zarr/zarr-in-practice.ipynb

```diff
@@ -8,11 +8,38 @@
     "\n",
```

Comment:
<p>From the information above, we can see that the default compression, <code>zstd</code>, is a good choice for n-dimensional data.</p><p> </p><p>The <code>Blosc</code> compressor is a meta compressor, which means it implements multiple different internal compressors. Let's try it with <code>lz4</code> compression (this was the default compression for zarr-python&lt;3). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>From the diff it looks like <code>Blosc</code> was the default last time this notebook ran. My understanding is that in zarr-python&lt;3 the default depended on which libraries were available in the environment. So maybe <code>lz4</code> was the default if <code>Blosc</code> wasn't available.   </p>